warehouse/templates/manage/account.html

```diff
@@ -567,7 +567,7 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
         {% if event.additional.two_factor_method == None %}
           {% trans %}None{% endtrans %}
         {% elif event.additional.two_factor_method  == "webauthn" %}
-          {% trans %}Security device (<abbr title="web authentication">WebAuthn</abbr>){% endtrans %}
+          {% trans %}Security device (<abbr title="web authentication">WebAuthn</abbr>){% endtrans %}{% if event.additional.two_factor_label %} - "{{ event.additional.two_factor_label }}"{% endif %}
```

Comment:
```suggestion
          {% if event.additional.two_factor_label %}<strong>"{{ event.additional.two_factor_label }}"</strong> - {% endif %}{% trans %}Security device (<abbr title="web authentication">WebAuthn</abbr>){% endtrans %}
```

To make this consistent with https://github.com/pypa/warehouse/blob/ce795ed0f38b0aa093df2d5bb7280210b5c7b1a2/warehouse/templates/manage/account.html#L475